src/components/avatar/Avatar.tsx

```diff
@@ -67,13 +86,17 @@ function Avatar({ avatarUrl, className, name, id, isExternal, shouldShowExternal
         avatar = <UnknownUserAvatar className="avatar-icon" />;
     }
 
+    let badge = null;
+    if (shouldShowExternal && isExternal) {
+        badge = <GlobeBadge16 className="bdl-Avatar-badge bdl-Avatar-externalBadge" />;
+    } else if (badgeIcon) {
+        badge = React.cloneElement(badgeIcon, {
```

Comment:
Rather than cloning the element, can we add another wrapping div around it and apply the styles there?

Reply:
Yes, we can. I usually try to avoid creating `div`'s cause too many of them will mage page slow.